### ISSUE TITLE
Deploy Tool: Error Deploying syslog .so files

### ISSUE THREAD

**Issue by dbendele** [LabVIEW-DCAF/TagEditorCore#310]
Error on Deploy when multiple versions of LabVIEW present, but one of them does not have LabVIEW Real Time installed.
![image](https://cloud.githubusercontent.com/assets/25972098/26263329/06c3a590-3c9e-11e7-8616-941328d47737.png)

**Comment by Beazurt** [comment]
Do we have a root cause for why this is happening? I can't think of any reason why it would. Does the deploy tool introduce LabVIEW RT as a dependency of DCAF?

**Comment by pollockm** [comment]
...I would believe this could be caused if multiple LV versions are present, but the LV version in use did not have the syslog wrapper dependency installed.  I don't see how RT would affect, since the code uses the vi.lib constant and does not reference the RT directories.

**Comment by Beazurt** [comment]
I just ran into this same issue. My dialog also showed that it was looking for the .so files in the LabVIEW 2015 folder which I no longer have installed (but there is still a legacy LabVIEW 2015 folder in program files that is mostly empty). I haven't investigated yet, but I would think the Deploy Tool should be looking for those files in the version of LabVIEW that is being used. In my case this was 2017.

**Comment by Beazurt** [comment]
When I created a new config file in 2017 it worked fine. When I used the shipping example config, created in 2013, I got the error above.

I am pretty sure the problem is that we create the default paths for these files in the version of LabVIEW that created the configuration file. However since this only happens once, if you try to deploy a configuration file in a version of the editor running a different version of LabVIEW, it doesn't work.
